lakefile.toml

```diff
@@ -13,6 +13,7 @@ linter.mathlibStandardSet = true
 [[require]]
 name = "mathlib"
 git = "https://github.com/leanprover-community/mathlib4.git"
+rev = "v4.22.0-rc2"
```

Comment:
Are you sure you want to land this?

Reply:
Done. Thanks.

Reply:
Hm... good question. In any case, this need not block landing this PR.